packages/terra-functional-testing/src/commands/utils/gh.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * This module encapsulates the full-power octokit library inside a simple interface.
+ * https://github.com/octokit/octokit.js
+ *
+ * It supports both github.com and enterprise github hosts.
+ * Requires your github host to have API v3 and https.
+ */
+const { Octokit } = require('octokit');
+
+/**
+ * Look up information about a github PR.
+ *
+ * @param {string} config.githubHostname The hostname of your github. Can be 'github.com' or your enterprise github (e.g. 'github.example.com').
+ * @param {string} config.repoOwner Would be 'cerner' for https://github.com/cerner/terra-core/pulls/1.
+ * @param {string} config.repoName Would be 'terra-core' for https://github.com/cerner/terra-core/pulls/1.
+ * @param {string} config.prNumber Would be '1' for https://github.com/cerner/terra-core/pulls/1.
+ * @returns An objection containing the PR's number, unique ID, head and base refs.
+ *
+ * If anything goes wrong looking up the PR, an exception will be thrown.
+ */
+async function getPr(config) {
```

Comment:
nitpicky but `getPr` or `getPR`? The latter seems more consistent with how we tend to abbreviate pull request into PR.